src/views/GovernoAberto.vue

```diff
@@ -6,6 +6,8 @@
 
 		<Indice :titulos="titulosLimpo"></Indice>
 
+		<router-view />
```

Comment:
vai manter isso na prod?

Reply:
quando é utilizado o children na página esse ``<router-view />`` é necessário para renderizar a view ThreadComments. Só mantei se for usar o thread comments.

Reply:
Como não afeta o conteúdo sugiro manter. @lordscorp, veja a rota abaixo como exemplo:
https://deploy-preview-201--participe-gestaourbana.netlify.com/governo-aberto/comentarios/1

É uma alteração no componente `CommentContext.vue` e não altera o conteúdo. 

@lordscorp tudo bem manter assim?